dask_cudf/tests/test_groupby.py

```diff
@@ -31,15 +33,19 @@ def test_groupby(func):
 
     ddf = dask_cudf.from_cudf(gdf, npartitions=5)
 
-    a = func(gdf).to_pandas()
-    b = func(ddf).compute().to_pandas()
+    a = pd.DataFrame(func(gdf).to_pandas())
+    b = pd.DataFrame(func(ddf).compute().to_pandas())
 
     a.index.name = None
     a.name = None
     b.index.name = None
     b.name = None
 
-    dd.assert_eq(a, b)
+    dd.assert_eq(
+        a,
+        b,
+        check_dtype=check_dtype
+    )
```

Comment:
Any chance we can just typecast from int32 to int64 in the case the aggregation is a count here? This is a lot of code that's not straightforward to capture that single case.

Reply:
The parameter is a lambda from which it's impossible to get information about the aggregation type. I'll split the test into two tests and parametrize the first by aggregation type